js-exercises/abbreviate-string/abbreviateString.js

```diff
@@ -0,0 +1,11 @@
+import { isString } from 'util';
+
+function abbreviateString(str) {
+  if (!isString(str) || !str.length) throw new Error(' Please use a valid string.');
+
+  const strArray = str.trim().split(' ');
+  const suffix = `${strArray[strArray.length - 1].charAt(0).toUpperCase()}`;
```

Comment:
Did you consider when the given string has only one word? First and last words would be the same, I believe.

Also, I don't see a reason to use template string here 🤔
Because whatever `strArray[strArray.length - 1].charAt(0).toUpperCase()` returns is already a string/charactor